jenkins/Jenkinsfile

```diff
@@ -28,6 +28,14 @@ pipeline {
                 '''
             }
         }
+        stage('setup spack') {
```

Comment:
Consider renaming this, since it's not only setting up spack but also installs the spack environement.